### DISCUSSION TITLE
Not working on LineageOs Android TV (KonstaKANG)

### DISCUSSION THREAD

**Discussion by Varrial** [aidanmacgregor/Netflix_ATV_L3_DRM_Uncertified_Mod#25] [General]
Hi there, I have rpi4b 8gb with android tv lineage os 20 from KonstaKANG. Security level is L3. I have tried to install downloaded app, but after installation I get error “make sure that your device has network connectivity and that the date and time settings are correct (-121)”. I have tried to change ESN number, but I don’t know how to find one or is this even a problem here. Also after first open of application, everything for a second looks fine, and then pops up error message, but background still looks like working one, I will attache a photo (sorry for censorship). Can you give me some advice please
![IMG_6034](https://github.com/aidanmacgregor/Netflix_ATV_L3_DRM_Uncertified_Mod/assets/73936910/a1c345b7-83df-444a-8a32-7b14b06b4d73)

**Comment by Varrial** [comment]
after re-opening app there is only black screen behind the error message

**Comment by ankusa** [comment]
https://github.com/aidanmacgregor/Netflix_ATV_L3_DRM_Uncertified_Mod#known-issues

Known Issues
Android 12+ (Internet connectivity issue)

**Comment by Varrial** [comment]
Im not sure, but Im guessing app have internet connectivity, that's why I see background image. I just need a way to get rid of that prompt.

**Comment by aidanmacgregor** [comment]
the app appears to get some connectivity like you describe, seems like a check somewhere and an API doesn't respond as expected to report (maybe data type EG. Wi-Fi or LTE)  not too sure but sounds like this or something similar, the logical conclusion :)

**Comment by Haas11** [comment]
confirm i see the same
